vnext/Microsoft.ReactNative.Cxx/ShareToMac.bat

```diff
@@ -0,0 +1,24 @@
+pushd .
+copy Crash.h                    ..\..\..\react-native-macos\RNTester\RNTester-macOS\TurboModuleCxx
```

Comment:
make ..\..\..\react-native-macos into a variable since not everyone will have the same clone setup

Reply:
After the development is done (should before the end of OCT), this file will be deleted. Files will be shared by npm packages so there is no need to do this at local machine.

So I think it is fine because only me is using this script.

Reply:
If so then there's no need to check it into the project repo? :)

Reply:
I decide to remove this bat file